bigbluebutton-html5/imports/api/2.0/meetings/server/modifiers/changeLockSettings.js

```diff
@@ -0,0 +1,23 @@
+import Logger from '/imports/startup/server/logger';
+import Meetings from '/imports/api/2.0/meetings';
+
+export default function changeLockSettings(meetingId, payload) {
+

```

Comment:
Please add a check for the payload like we have in https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/api/2.0/chat/server/modifiers/addChat.js#L32-L40